### PR TITLE
Add bootstrap

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/939
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/70
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/159

### CODE REVIEW

File: DisplayBundle/Resources/public/requireConf.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ require.config({
     baseUrl: '/bundles',
     paths: {
         openOrchestraCss: 'openorchestradisplay/libs/openOrchestraCss',
-        jquery: 'openorchestradisplay/libs/jquery-2.0.2.min'
+        jquery: 'openorchestradisplay/libs/jquery-2.0.2.min',
+        bootstrap: 'openorchestradisplay/libs/bootstrap/bootstrap.min'
```
Comment (itkg-alavieille) [inline DisplayBundle/Resources/public/requireConf.js]: why add bootstrap, jssor need it ?

Comment (itkg-ngilain) [inline DisplayBundle/Resources/public/requireConf.js]: yes it is required by the behavior chosen for the carousel block : http://www.jssor.com/bootstrap/bootstrap-slider.html

**Comment by itkg-alavieille** [comment]
you can use this carrousel without bootstrap 
http://www.jssor.com/demos/simple-fade-slideshow.html

**Comment by itkg-ngilain** [comment]
Ok I'll check if it's quick to remove the dependancy using that version

**Comment by itkg-ngilain** [comment]
After review, bootstrap was even not used in the features I used from that jssor version. So I can close this PR ans delete this branch.
TY @itkg-alavieille :)

**Pull request closed without merging**